### PR TITLE
Document the EventRateLimit admission controller

### LINKED ISSUE #6202: EventRateLimit admission controller not documented
**This is a...** 

- [X ] Feature Request
- [ ] Bug Report

**Problem:**
The EventRateLimit admission controller needs documentation.

**Proposed Solution:**
Document it, of course.

**Page to Update:**
http://kubernetes.io/...

### PR DESCRIPTION
**Pull request by tengqm**
Closes #6202

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6203)

### CODE REVIEW

**Comment by tengqm** [comment]
@staebler

**Comment by zacharysarah** [comment]
Updated OP from "fixed" to "closes" to automate issue closure when this PR merges.

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -100,6 +100,42 @@ If your cluster supports containers that run with escalated privileges, and you
...
+ * Enabling the `EventRateLimit` admission controller;
+ * Including a `EventRateLimit` configuration in the file provided to the API
+   server's command line flag `--admission-control-config-file`.
+
+There are four types of limits that can be specified in the configuration:
+
+ * `Server`: all event requests received by the API server share a single bucket;
+ * `Namespace`: each namespace has a dedicated bucket;
+ * `User`: each user is allocated a bucket;
+ * `SourceAndObject`: a bucket is assigned by each combination of source and
```
Comment (steveperry-53) [inline docs/admin/admission-controllers.md]: Suggestion:
For this list, start each sentence with a cap and end with a period. That would be closer to the style we've been using. Example:

Server: All event requests received by the API server share a single bucket.

**Comment by steveperry-53** [comment]
@tengqm One comment. Otherwise docs LGTM.

**Pull request merged**